### ISSUE TITLE
Model build failure on startup

### ISSUE THREAD

**Issue by biennvops** [0seba/VoxCPMANE#6]
```bash
uv run voxcpmane-server --help
🚀 Downloading/loading model files from Hugging Face Hub repo: seba/VoxCPM1.5-ANE
Fetching 96 files: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 96/96 [00:00<00:00, 17515.04it/s]
Download complete: : 0.00B [00:00, ?B/s]              Loading CoreML models...                                                                          | 0/96 [00:00<?, ?it/s]
Download complete: : 0.00B [00:06, ?B/s]
voxcpm_model_path: openbmb/VoxCPM-0.5B, zipenhancer_model_path: iic/speech_zipenhancer_ans_multiloss_16k_base, enable_denoiser: False
Warning: You are sending unauthenticated requests to the HF Hub. Please set a HF_TOKEN to enable higher rate limits and faster downloads.
❌ An unexpected error occurred during model setup: {
    NSLocalizedDescription = "Failed to build the model execution plan using a model architecture file '/Users/user/.cache/huggingface/hub/models--seba--VoxCPM1.5-ANE/snapshots/e8842decd18875ec71d0f2e903b825de84c37483/base_lm_mf_f16.mlmodelc/model.mil' with error code: -14.";
}
Traceback (most recent call last):
  File "/Users/user/Developer/voxcpmane/.venv/bin/voxcpmane-server", line 4, in <module>
    from voxcpmane.server import main
  File "/Users/user/Developer/voxcpmane/.venv/lib/python3.11/site-packages/voxcpmane/server.py", line 119, in <module>
    model = VoxCPMANE(
            ^^^^^^^^^^
  File "/Users/user/Developer/voxcpmane/.venv/lib/python3.11/site-packages/voxcpmane/voxcpm.py", line 1567, in __init__
    self.tts_model = VoxCPMModelANE(
                     ^^^^^^^^^^^^^^^
  File "/Users/user/Developer/voxcpmane/.venv/lib/python3.11/site-packages/voxcpmane/voxcpm.py", line 591, in __init__
    self.load_preprocessing_models()
  File "/Users/user/Developer/voxcpmane/.venv/lib/python3.11/site-packages/voxcpmane/voxcpm.py", line 1235, in load_preprocessing_models
    base_lm = ct.models.CompiledMLModel(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/Developer/voxcpmane/.venv/lib/python3.11/site-packages/coremltools/models/_compiled_model.py", line 127, in __init__
    self._proxy = _MLModelProxy(
                  ^^^^^^^^^^^^^^
RuntimeError: {
    NSLocalizedDescription = "Failed to build the model execution plan using a model architecture file '/Users/user/.cache/huggingface/hub/models--seba--VoxCPM1.5-ANE/snapshots/e8842decd18875ec71d0f2e903b825de84c37483/base_lm_mf_f16.mlmodelc/model.mil' with error code: -14.";
}
```

This is with the beta version. Stable didn't work either, albeit different log? I didn't get the log for it however

**Comment by 0seba** [comment]
Hi, sorry for the delay I was away from my computer. Which OS are you running on? I rely on features available from macOS 15

**Comment by biennvops** [comment]
@0seba I'm on macOS Tahoe 26.2.

**Comment by 0seba** [comment]
Just for testing, could you install from source and modify that line 1236 of the voxcpm.py file to use `ct.ComputeUnit.CPU_ONLY` or `ct.ComputeUnit.CPU_AND_GPU` as `compute_units` instead of the neural engine

**Comment by biennvops** [comment]
Hmm, I got a different error by running the build from source code.
```bash
❌ An unexpected error occurred during model setup: {
    NSLocalizedDescription = "`MLModelConfiguration`'s `.functionName` property must be `nil` unless the model type is ML Program.";
}
Traceback (most recent call last):
  File "/Users/user/Developer/VoxCPMANE/.venv/bin/voxcpmane-server", line 4, in <module>
    from voxcpmane.server import main
  File "/Users/user/Developer/VoxCPMANE/src/voxcpmane/server.py", line 118, in <module>
    model = VoxCPMANE(
            ^^^^^^^^^^
  File "/Users/user/Developer/VoxCPMANE/src/voxcpmane/voxcpm.py", line 1519, in __init__
    self.tts_model = VoxCPMModelANE.from_local(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/Developer/VoxCPMANE/src/voxcpmane/voxcpm.py", line 1142, in from_local
    model = cls(
            ^^^^
  File "/Users/user/Developer/VoxCPMANE/src/voxcpmane/voxcpm.py", line 584, in __init__
    self.load_preprocessing_models()
  File "/Users/user/Developer/VoxCPMANE/src/voxcpmane/voxcpm.py", line 1213, in load_preprocessing_models
    base_lm = ct.models.CompiledMLModel(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/Developer/VoxCPMANE/.venv/lib/python3.11/site-packages/coremltools/models/_compiled_model.py", line 127, in __init__
    self._proxy = _MLModelProxy(
                  ^^^^^^^^^^^^^^
RuntimeError: {
    NSLocalizedDescription = "`MLModelConfiguration`'s `.functionName` property must be `nil` unless the model type is ML Program.";
}
```
Tried both `CPU_ONLY` and `CPU_AND_GPU`.
